concrete/src/Entity/File/File.php

```diff
@@ -496,6 +501,41 @@ public function getFileID()
         return $this->fID;
     }
 
+    /**
+     * File UUID.
+     *
+     * @return string
+     */
+    public function getFileUUID()
+    {
+        return $this->fUUID;
+    }
+
+    public function generateFileUUID()
```

Comment:
Do we want this or should we integrate something like https://packagist.org/packages/ramsey/uuid ?

Reply:
done.

Reply:
What does that ramsey package offer more than https://packagist.org/packages/symfony/polyfill-uuid? The symfony package has less dependencies...

Reply:
Ramsey uuid is more feature rich and more popular vs. symfony uuid has less dependencies and is a little bit faster in generation, see: https://symfony.com/blog/introducing-the-new-symfony-uuid-polyfill

The requirements for uuid generation in concrete5 is very minimal so symfony uuid is enough.